config/confighttp/README.md

```diff
@@ -26,6 +26,27 @@ README](../configtls/README.md).
 - `compression`: Compression type to use among `gzip`, `zstd`, `snappy`, `zlib`, and `deflate`.
   - look at the documentation for the server-side of the communication.
   - `none` will be treated as uncompressed, and any other inputs will cause an error.
+  - Compression levels can now be configured as part of the compression type like below. Not specifying any compression level will result in the default.
```

Comment:
`snappy` should be mentioned as well, without levels (we should fail for snappy/3)